### PR TITLE
Panel: Stretch external panel to document height

### LINKED ISSUE #7983: External panel not height 100%
Hi

I run demo: http://demos.jquerymobile.com/1.4.2/panel-external/
Rotate horizontal, click button Overlay to show panel, panel not full height

untitled

### PR DESCRIPTION
**Pull request by gabrielschulhof**
Fixes gh-7983

### CODE REVIEW

File: js/widgets/panel.js [hunk 1]
```diff
@@ -169,6 +169,10 @@ $.widget( "mobile.panel", {
 			if ( expand ) {
 				self._unfixPanel();
 				$.mobile.resetActivePageHeight( panelInnerHeight );
+			} else if ( !this._parentPage ) {
+				if ( this.element.height() < this.document.height() ) {
+					this.element.height( this.document.height() );
```
Comment (arschmitz) [inline js/widgets/panel.js]: Do we really want height here? or should this be `outerHeight`?

Comment (gabrielschulhof) [inline js/widgets/panel.js]: @jaspermdegroot You're the expert here. What do you think?

Comment (jaspermdegroot) [inline js/widgets/panel.js]: @gabrielschulhof - It should be ```outerHeight```.

**Comment by arschmitz** [comment]
:+1:

**Pull request closed without merging**